gemma-core/src/main/java/ubic/gemma/core/analysis/report/ExpressionExperimentReportServiceImpl.java

```diff
@@ -126,7 +126,7 @@ public ExpressionExperimentDetailsValueObject generateSummary( Long id ) {
     public void generateSummaryObjects() {
         Collection<Long> ids = EntityUtils.getIds( expressionExperimentService.loadAll() );
         Collection<ExpressionExperimentDetailsValueObject> vos = expressionExperimentService
-                .loadDetailsValueObjects( null, false, ids, null, 0, 0 );
+                .loadDetailsValueObjects( null, ids, null, 0, 0 );
```

Comment:
We use `Sort` instead of passing the property and direction as separate argument. Few advantages:

 - if sort is null, no sort is performed
 - if the direction is null, the default direction is used
 - less confusion about booleans (we used to ascending and descending argument)